### DISCUSSION TITLE
Figure size error during ploting

### DISCUSSION THREAD

**Discussion by nouman073** [AammarTufail/python_ka_chilla2023#55] [Q&A]
kashti.survived.value_counts().plot(kind="bar") i run this line of code . it did not show the plot. how to solve it

showing this error. <Figure size 640x480 with 1 Axes>

**Comment by saleharashid01** [comment]
try this

![hwo](https://user-images.githubusercontent.com/121869148/211894923-eaec00d6-22dc-43b3-98e4-4809130a8f58.jpg)
